documentation/collectionruleexamples.md

```diff
@@ -0,0 +1,223 @@
+# Collection Rule Examples
+
+The following examples provide sample scenarios for using a collection rule. These templates can be copied directly into your configuration file with minimal adjustments to work with your application, or they can be adjusted for your specific use-case. [Learn more about configuring collection rules](collectionrules.md).
+
+## Collect GCDump - Startup (`Startup` Trigger)
+
+### JSON
+
+```json
+{
+  "GCDumpOnStartup": {
```

Comment:
One possibly useful scenario for startup is tracing assembly loads. See https://docs.microsoft.com/en-us/dotnet/fundamentals/diagnostics/runtime-loader-binder-events.

A good example for gcdump might be an event counter trigger against the heap size counter (https://docs.microsoft.com/en-us/dotnet/core/diagnostics/available-counters)